### PR TITLE
Android: Dark theme improvements for newsletters.

### PR DESCRIPTION
**Pull request by manderson23**
Improves readability of HTML heavy newsletters in dark theme.

### CODE REVIEW

File: clients/android/NewsBlur/assets/dark_reading.css [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 }
 
-code {
+pre, blockquote, code {
     background-color: #4C4C4C;
 }
 
-pre, blockquote {
-    background-color: #4C4C4C;
+blockquote > p, pre > span, code span {
```
Comment (samuelclay) [inline clients/android/NewsBlur/assets/dark_reading.css]: What about just `pre`?

Comment (manderson23) [inline clients/android/NewsBlur/assets/dark_reading.css]: pre is now handled in the previous selector.

Comment (samuelclay) [inline clients/android/NewsBlur/assets/dark_reading.css]: Doh, I should have noticed that. Just got back form NYC and I'm at half-functioning.

Comment (samuelclay) [inline clients/android/NewsBlur/assets/dark_reading.css]: Can you give me examples of the newsletters fixed by these selectors? I need to also implement this on iOS and possibly web.

Comment (manderson23) [inline clients/android/NewsBlur/assets/dark_reading.css]: http://www.nytimes.com/newsletters/what-were-reading?nlid=72249677 - still
needs a few tweaks. Title text is still black for example.

http://softwareleadweekly.com/

Pocket hits - not sure there is a link without being signed in

http://www.mountaingoatsoftware.com/ - weekly e-mail tips

http://www.longreads.com/ - weekly e-mail. This is the one the had inline
!important do doesn't look good at all for now.

**Pull request merged**